### ISSUE TITLE
Removing slingring not possible if installation failed due to missing dependencies

### ISSUE THREAD

**Issue by michaelriedel** [vlow/slingring#2]
Tried to install slingring on an Ubuntu system (actually the Win10 linux subsystem) without having all of the required dependencies installed:
```
miri@gpc:/mnt/c/Users/Carmen/Downloads/slingring-master$ sudo dpkg -i slingring_0.8.2_all.deb
Vormals nicht ausgewähltes Paket slingring wird gewählt.
(Lese Datenbank ... 31981 Dateien und Verzeichnisse sind derzeit installiert.)
Vorbereitung zum Entpacken von slingring_0.8.2_all.deb ...
Entpacken von slingring (0.8.2) ...
dpkg: Abhängigkeitsprobleme verhindern Konfiguration von slingring:
 slingring hängt ab von python3-yaml; aber:
  Paket python3-yaml ist nicht installiert.
 slingring hängt ab von python3-jinja2; aber:
  Paket python3-jinja2 ist nicht installiert.

dpkg: Fehler beim Bearbeiten des Paketes slingring (--install):
 Abhängigkeitsprobleme - verbleibt unkonfiguriert
Fehler traten auf beim Bearbeiten von:
 slingring
```

After that, slingring seems to be installed (but not configured),  but cannot be uninstalled:
```
miri@gpc:/mnt/c/Users/Carmen/Downloads/slingring-master$ sudo dpkg -r slingring
(Lese Datenbank ... 32087 Dateien und Verzeichnisse sind derzeit installiert.)
Entfernen von slingring (0.8.2) ...
rm: das Entfernen von »/usr/bin/seed“ ist nicht möglich: Datei oder Verzeichnis nicht gefunden
rm: das Entfernen von »/usr/bin/universe“ ist nicht möglich: Datei oder Verzeichnis nicht gefunden
dpkg: Fehler beim Bearbeiten des Paketes slingring (--remove):
 Unterprozess installiertes post-removal-Skript gab den Fehlerwert 1 zurück
Fehler traten auf beim Bearbeiten von:
 slingring
```

I created the two files that the uninstall script tried to remove:
```
miri@gpc:/mnt/c/Users/Carmen/Downloads/slingring-master$ sudo touch /usr/bin/seed
miri@gpc:/mnt/c/Users/Carmen/Downloads/slingring-master$ sudo touch /usr/bin/universe
```
..and after that I was able to uninstall slingring again.

Maybe the uninstall script should check for the existance of the files prior to deleting them?

(Possibly, the missing dependencies could have been installed by executing `apt-get -f install`, instead of trying to remove slingring and install the dependencies before re-installing it; nevertheless, I think I should be able to remove a package that is installed but not yet configured).

**Comment by vlow** [comment]
Installing using dpkg seems to ignore the dependencies. Since the two missing files are actually symlinks created by the create-symlinks.sh script, it looks like this script has not been invoked when the installation failed. As you mentioned, the fix is rather straight forward: the post-removal script should only remove the symlinks if they have been created. I will fix this.

Regarding the failing installation: Did you try installing slingring using apt? This should automatically resolve all dependencies.

**Comment by vlow** [comment]
fixed in 93d51972
